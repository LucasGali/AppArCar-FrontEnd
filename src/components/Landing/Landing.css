.landing-section {
  height: 100%;
  display: flex;
  padding: 0;
  flex-direction: column;
}

.landing-section h1 {
  align-self: center;
}

.landing-section h1::after {
  border-bottom: solid 2px #287ef4;
  width: 70px;
  content: '';
  display: block;
  position: relative;
  left: 0;
  right: 0;
  margin: 30px auto 35px;
}

.landing-section p {
  margin: 20px;
  align-self: center;
}

@media only screen and (max-width: 500px) and (max-height: 850px) {
  .landing-section {
    width: 100vw;
    height: 45vh;
  }
}

@media only screen and (max-width: 850px) {
  .landing-section {
    height: 29vh;
    margin: 5px;
  }
}
