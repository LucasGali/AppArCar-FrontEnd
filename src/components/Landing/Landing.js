import React from 'react'
import './Landing.css'

const Landing = (props) => {
  return (
    <div className="landing-section">
      <h1>{props.name}</h1>
      <p>
        {props.text}
      </p>
      <p>
        {props.text2}
      </p>
    </div>
  );
};

export default Landing
