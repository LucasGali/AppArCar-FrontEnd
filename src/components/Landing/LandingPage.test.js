import { render, screen } from '@testing-library/react';
import Landing from './Landing';
import { MemoryRouter } from 'react-router-dom';

describe('LandingSection', () => {
  it('renders default form according to snapshot', () => {
    const { asFragment } = render(
      <MemoryRouter>
        <Landing />
      </MemoryRouter>,
    );
    expect(asFragment(<Landing />)).toMatchSnapshot();
  });
});
