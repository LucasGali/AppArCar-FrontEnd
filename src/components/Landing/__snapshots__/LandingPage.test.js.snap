// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingSection renders default form according to snapshot 1`] = `
<DocumentFragment>
  <div
    class="landing-section"
  >
    <h1 />
    <p />
    <p />
  </div>
</DocumentFragment>
`;
